runlnxlink.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+lnxlink -ic ~/Documents/lnxlink.yaml
```

Comment:
how do you expect this file in `~/Documents` to exist?

Reply:
This is automatically created by the application and there are steps to configure it.